src/operator/src/req_convert/insert/table_to_region.rs

```diff
@@ -44,7 +45,19 @@ impl<'a> TableToRegion<'a> {
         let requests = Partitioner::new(self.partition_manager)
             .partition_insert_requests(self.table_info.table_id(), rows)
             .await?;
-        Ok(RegionInsertRequests { requests })
+
+        let requests = RegionInsertRequests { requests };
+        if is_ttl_imme_table(self.table_info) {
```

Comment:
I think `is_ttl_imme_table` should be renamed too, and can be moved into `TableInfo` struct.